analyses/mdg/urban_exploration.md

```diff
@@ -0,0 +1,59 @@
+### Urban classification in Madagascar
+
+Very simple script to classify ADM4 areas in Madagascar as urban based on GHS data.
+
+```python
+from pathlib import Path
+import sys
+import os
+import geopandas as gpd
+import rasterio
+import numpy as np
+
+path_mod = f"{Path(os.path.dirname(os.path.abspath(''))).parents[0]}/"
+sys.path.append(path_mod)
+
+from src.indicators.drought.config import Config
+from src.utils_general.ghs import get_ghs_data
+from src.utils_general.ghs import classify_urban_areas
+
+# Setup
+
+iso3 = "mdg"
+raw_dir = os.path.join(os.environ["AA_DATA_DIR"], 'public', 'raw', iso3)
+processed_dir = os.path.join(os.environ["AA_DATA_DIR"], 'public', 'processed', iso3)
+adm4_path = os.path.join(raw_dir, 'cod_ab', 'mdg_admbnda_adm4_BNGRC_OCHA_20181031.shp')
```

Comment:
would be nice to define the admin level as a variable instead

Reply:
Yes, done!